src/skeleton/type.js

```diff
@@ -0,0 +1,17 @@
+ import * as React from 'react';
+import type {OverrideT} from '../helpers/overrides.js';
+
+
+
+
+export type SkeletonPropsT = {
+
+  overrides?: OverrideT,
+
+  children?: React.Node,
+
+  rows: number,
+
+  name?: string
```

Comment:
For overrides, see this example as a reference: https://github.com/uber/baseweb/blob/master/src/button/types.js#L13-L20

It should be something like:
```js
overrides?: {|
  Root?: OverrideT,
  Row?: OverrideT,
|}
```

What purpose do the `children` and `name` props serve?

Reply:
Modified.

Reply:
I also delete these two prop. It seems no need for now.

Reply:
Modify the override.